test/test_suite.py

```diff
@@ -107,19 +109,130 @@ def define_temp_tables(cls, metadata):
                 DDL("drop view user_tmp_v")
             )
 
-    @skip_if('hana')
-    @testing.requires.temp_table_reflection
-    @testing.requires.unique_constraint_reflection
-    def test_get_temp_table_unique_constraints(self):
```

Comment:
We dont need it anymore because this test is a part of the SQLAlchemy dialect compliance suite and I just added this test in the SQLAlchemy-hana test suite so that we can skip the test, because until then the unique constraint reflection wasnt working properly and skipping the test was better than letting it fail. I already fixed the unique constraint reflection problem and hence there is no need to skip this test and we can let the test run normally from the SQLAlchemy test suite.